Libplanet.Net/Consensus/ConsensusContext.cs

```diff
@@ -333,7 +333,7 @@ private void OnBlockChainTipChanged(object? sender, (Block<T> OldTip, Block<T> N
                 _newHeightCts.Token);
         }
 
-        private void ClearOldLastCommitCache(long maxSize = 30)
+        private void ClearOldLastCommitCache(long currentHeight, long maxSize = 30)
```

Comment:
Is this kind of fine control really necessary? I think referencing `_blockChain.Tip.Index` directly might be better. 😶

Reply:
I thought there would be a case if a block has appended while in `ClearOldLastCommitCache`, then exclude target can be changed if `_blockChain.Tip.Index - 1` is used. On second thought, It has no problem doing so due to the lack of `LastCommit` of the synced block. Still, I don't see any advantage of using `_blockChain.Tip.Index`.

Reply:
In general, additional parameter means the caller needs to be aware of the wider context in which the method should be called. Specifically, the caller needs to know not only "when" but also "how" this method should be called. I don't think that kind of logical burdening is necessary to the outside caller.